projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.js

```diff
@@ -49,11 +51,20 @@ export default function Player( { isSelected, attributes } ) {
 	const loadingViewStyle = style[ 'videopress-player__loading' ];
 
 	const [ isPlayerLoaded, setIsPlayerLoaded ] = useState( false );
-	const [ isPlayerReady, setIsPlayerReady ] = useState( false );
+	const [ isPlayerReady, setIsPlayerReady ] = useState( IS_ANDROID );
```

Comment:
We also need to update this line to avoid setting `isPlayerReady` to `false` when the video changes:
https://github.com/Automattic/jetpack/blob/4905b22b180d71301062f4d41479494997c87976/projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.js#L81

```patch
diff --git forkSrcPrefix/projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.js forkDstPrefix/projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.js
index cf5ddf569ffa5edf8c8854234b27d79224a0a60f..a539df56ea5b12e74b7f85606d2c0693541f6731 100644
--- forkSrcPrefix/projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.js
+++ forkDstPrefix/projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.js
@@ -78,7 +78,7 @@ export default function Player( { isSelected, attributes } ) {
 	useEffect( () => {
 		if ( guid ) {
 			setIsPlayerLoaded( false );
-			setIsPlayerReady( false );
+			setIsPlayerReady( IS_ANDROID );
 		}
 	}, [ guid ] );
 

```